crates/api_models/src/payments.rs

```diff
@@ -978,56 +1003,75 @@ pub struct Metadata {
     pub data: serde_json::Value,
 }
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 pub struct PaymentsSessionRequest {
+    /// The identifier for the payment
     pub payment_id: String,
+    /// This is a 15 minute expiry token which shall be used from the client to authenticate and perform sessions from the SDK
     pub client_secret: String,
+    /// The list of the supported wallets
     pub wallets: Vec<api_enums::SupportedWallets>,
 }
 
-#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct GpayAllowedMethodsParameters {
+    /// The list of allowed auth methods (ex: 3DS, No3DS, PAN_ONLY etc)
     pub allowed_auth_methods: Vec<String>,
+    /// The list of allowed card networks (ex: AMEX,JCB etc)
     pub allowed_card_networks: Vec<String>,
 }
 
-#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct GpayTokenParameters {
+    /// The name of the connector
     pub gateway: String,
+    /// The merchant ID registered in the connector associated
     pub gateway_merchant_id: String,
 }
 
-#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct GpayTokenizationSpecification {
+    /// The token specification type(ex: PAYMENT_GATEWAY)
     #[serde(rename = "type")]
     pub token_specification_type: String,
+    /// The parameters for the token specification gpay
     pub parameters: GpayTokenParameters,
 }
 
-#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct GpayAllowedPaymentMethods {
+    /// The type of payment method
     #[serde(rename = "type")]
     pub payment_method_type: String,
+    /// The parameters gpay requires
     pub parameters: GpayAllowedMethodsParameters,
+    /// The tokenization specification for gpay
     pub tokenization_specification: GpayTokenizationSpecification,
 }
 
-#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct GpayTransactionInfo {
+    /// The country code
     pub country_code: String,
+    /// The currency code
     pub currency_code: String,
+    /// The total price status (Ex: 'FINAL')
     pub total_price_status: String,
```

Comment:
You've used `ex` in some places to denote examples and `Ex` elsewhere. Please try to keep it uniform.